sw/device/silicon_creator/mask_rom/primitive_bootstrap.c

```diff
@@ -89,19 +92,49 @@ static rom_error_t bootstrap_requested(bool *result) {
   return kErrorOk;
 }
 
+/**
+ * Check that flash data partitions are all blank.
+ */
+bool flash_is_empty(void) {
```

Comment:
maybe this should be a testutils? since it is defined in multiple places?

Reply:
The second place is the mask ROM, so eventually, the DIF-based testutils wouldn't be able to be used there anyway. Meanwhile, I figured this wasn't a great function to use elsewhere, especially in the simulator.

I marked it `static`, so it's clear that it isn't meant to be exported. With no second user planned, shall we just leave it there?

Reply:
sgtm